File: src/main/java/fr/fantomitechno/dinnermod/DinnerMod.java
package fr.fantomitechno.dinnermod;

import net.fabricmc.api.ModInitializer;

import org.apache.logging.log4j.LogManager;

public class DinnerMod implements ModInitializer {

    @Override
    public void onInitialize() {
        LogManager.getLogger("dinnermod").info("DinnerMod succefuly loaded");
    }
}

File: src/main/kotlin/fr/fantomitechno/dinnermod/DinnerMod.kt
package fr.fantomitechno.dinnermod
import net.fabricmc.api.ModInitializer


@Suppress("UNUSED")
object DinnerMod: ModInitializer {
    private const val MOD_ID = "dinnermod"
    override fun onInitialize() {
        println("DinnerMod succefuly loaded")
    }
}